include/openssl/cms.h.in

```diff
@@ -397,6 +399,11 @@ int CMS_RecipientInfo_kari_decrypt(CMS_ContentInfo *cms,
 
 int CMS_SharedInfo_encode(unsigned char **pder, X509_ALGOR *kekalg,
                           ASN1_OCTET_STRING *ukm, int keylen);
+CMS_EnvelopedData *CMS_sign_encrypt(BIO *data, X509 *sign_cert, STACK_OF(X509) *certs,
+                                    EVP_PKEY *sign_key, unsigned int sign_flags,
+                                    STACK_OF(X509) *enc_recip, const EVP_CIPHER *cipher,
+                                    unsigned int enc_flags, OSSL_LIB_CTX *libctx,
+                                    const char *propq);
```

Comment:
According to https://github.com/openssl/openssl/issues/25491#issuecomment-2363546721 please make this function internal, 
renaming it to, e.g., `ossl_cms_sign_encrypt()`, 
moving its decls to, e.g., `include/internal/cryptlib.h` 
and moving its doc to  somewhere in ` doc/internal/man3/`

Reply:
added new file `include/internal/cms.h` instead of adding to `include/internal/cryptlib.h`, looks more cleaner.
 added new file `doc/internal/man3/ossl_cms_sign_encrypt.pod` for doc.